<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cell_Main</class>
 <widget class="QMainWindow" name="Cell_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[name = &quot;btn&quot;]{border-radius:4px;background: #409eff; color:white;font -size:14px;min-height: 28px; min-width: 76px;padding:4px}
QPushButton[name = &quot;btn&quot;]:pressed {background: #3a8ee6;}

QLabel[name=&quot;title&quot;]{font:30px '黑体';}
QLabel[name=&quot;lb&quot;]{font:14px '楷体';}
QLineEdit[name = &quot;le&quot;]{border-radius:4px;min-height:28px;min-width:280px; max-width :280px; border:1px solid black}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#tool{background-color: rgb(84,92,100);}
QWidget#tool QToolButton{border:none; background-color: rgb(84,92,100); color: white;min-height:55px; font:14px ‘黑体’; padding-left:10px}
QWidget#tool QToolButton:checked{background:rgb(67,74,80);}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="tool" native="true">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="btn_user">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>用户管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/ic_badge1.svg</normaloff>:/ic_badge1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_book">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>图书管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/ic_badge1.svg</normaloff>:/ic_badge1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btn_record">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>借阅记录</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/ic_badge1.svg</normaloff>:/ic_badge1.svg</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
